backend/src/Designer/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -759,6 +760,24 @@ public Stream GetProcessDefinitionFile()
             return OpenStreamByRelativePath(ProcessDefinitionFilePath);
         }
 
+        /// <summary>
+        /// Gets specified image from App/wwwroot/images folder of repo
+        /// </summary>
+        /// <param name="imageFilePath">The file name of the image</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> that observes if operation is cancelled.</param>
+        /// <returns>The image as stream</returns>
+        public Stream GetImage(string imageFilePath, CancellationToken cancellationToken = default)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            string imagePath = GetPathToImage(imageFilePath);
+            if (FileExistsByRelativePath(imagePath))
+            {
+                return OpenStreamByRelativePath(imagePath);
+            }
+
+            throw new FileNotFoundException("Image not found.");
```

Comment:
I think opening stream for a file that doens't exists will Throw FileNotFoundException so this is not needed. You can test it when adding a tests for image that doesn't exists.